Lib/idlelib/autocomplete_w.py

```diff
@@ -24,6 +29,13 @@
 WINCONFIG_SEQUENCE = "<Configure>"
 DOUBLECLICK_SEQUENCE = "<B1-Double-ButtonRelease>"
 
+
+_quote_re = re.compile(r"""["']""")
+def _find_first_quote(s):
```

Comment:
```suggestion
def _find_first_quote(quote):
```

Reply:
Thanks for the suggestion! I've done this, and otherwise improved the function e.g. adding a doc-string.

Reply:
Thanks! Doc-strings are always a good idea to include.